src/openforms/js/components/admin/form_design/registrations/objectsapi/fields/ObjectTypeVersionSelect.js

```diff
@@ -61,19 +61,14 @@ const ObjectTypeVersionSelect = () => {
             defaultMessage="Version"
           />
         }
+        noManageChildProps
       >
-        <Select
+        <ReactSelect
+          name="objecttypeVersion"
           required
-          disabled={!objecttype}
-          choices={choices}
-          id="id_objecttypeVersion"
-          {...fieldProps}
-          onChange={event => {
-            // overridden to handle the proper data type, since every <option value>
-            // turns into a string in HTML
-            const newVersion = parseInt(event.currentTarget.value, 10);
-            setValue(newVersion);
-          }}
+          options={options}
```

Comment:
Locally for me it seems that the version select is overlapped by the fieldset below it
![Screenshot from 2024-08-06 12-41-11](https://github.com/user-attachments/assets/30f17d37-3fc8-451c-9a0a-5187f163ba9a)

Reply:
I ran into it in another PR where it's fixed - django's `.form-row` hides overflow: https://github.com/open-formulieren/open-forms/pull/4586/files#diff-a95d74cf85e72c9e17c4281231439a672cb7d4d77646f77889f0e77499b76cfcR14

I'll cherry-pick that CSS fix to this branch :)